src/components/Search/SearchRouter/SearchRouterModal.tsx

```diff
@@ -7,9 +7,13 @@ import CONST from '@src/CONST';
 import SearchRouter from './SearchRouter';
 import {useSearchRouterContext} from './SearchRouterContext';
 
+const isMobileSafari = Browser.isMobileSafari();
+
 function SearchRouterModal() {
     const {shouldUseNarrowLayout} = useResponsiveLayout();
     const {isSearchRouterDisplayed, closeSearchRouter} = useSearchRouterContext();
+    // On mWeb Safari, the input caret stuck for a moment while the modal is animating. So, we hide the caret until the animation is done.
+    const [shouldHideInputCaret, setShouldHideInputCaret] = useState(isMobileSafari);
```

Comment:
Our style guides usually mention to leave empty lines before comments, so just a small suggestion

```suggestion

    // On mWeb Safari, the input caret stuck for a moment while the modal is animating. So, we hide the caret until the animation is done.
    const [shouldHideInputCaret, setShouldHideInputCaret] = useState(isMobileSafari);
```

Reply:
Updated